templates/named.zone.erb

```diff
@@ -23,4 +23,7 @@ zone "<%= @zone %>" {
 <% unless @forwarders.empty? -%>
     forwarders { <%= @forwarders.join('; ') %>; };
 <% end -%>
+<% if @_dns_notify -%>
```

Comment:
This should work well for undef, but it could write an empty string if set to `''` I think.  Perhaps a `validate_re` in the manifest would catch that.